imgui-core/manifest

```diff
@@ -1,13 +1,10 @@
 : 1
-name: imgui
+name: imgui-core
 version: 1.86.0-a.0.z
```

Comment:
Don't forget to remove the alpha part of the version, as this is a package it's ok to use the actual version (not alpha nor beta except if the upstream version is) and once published it can be revised.

Same for all other manifest files.

Reply:
I think using the alpha version while things are still in flux is a good idea. It's not ideal to make major changes/restructuring as revisions.

Reply:
Agree with @boris-kolpackov. I would like to keep the alpha version as long as we do not plan to release the package is on cppget.org. Which will anyway only happen when build2 0.15 is released officially.

Reply:
Yes, I meant "don't forget to change the version before publishing ...", no problem right now.